governance/executive-council.md

```diff
@@ -0,0 +1,261 @@
+- **Mission, vision, and values**: Decisions related to pyOpenSci’s core principles.
+- **Governance and policy changes**: Approval of changes to governance structures,
+  including updates to the Code of Conduct.
+- **Significant budget deviations**: Decisions involving major deviations from the
+  approved annual budget.
+- **Financial health implications**: Any decision with substantial implications for
+  pyOpenSci’s financial stability.
+- **Strategic direction**: Approval of major changes to organizational strategy and
+  development roadmaps.
+- **Executive director’s salary**: The EC independently determines the salary of the
+  Executive Director.
+
+:::note
+The Executive Director is authorized to make all other decisions. However, the
+Executive Council serves as a resource for the Executive Director and may provide
+feedback and input upon request.
+:::
+
+### Decision-making process
+
+The topics above require a motion and a vote for approval. Motions and approval can be made in Executive Council meetings or asynchronously via email or Slack. Motions and votes will be noted in the EC meeting minutes.
+
+:::{important}
+All decisions made by the Executive Council must align with the legal
+obligations and policies of pyOpenSci’s fiscal sponsor—[Community Initiatives](https://communityin.org/).
+:::
+
+## Collaboration between the Executive Director and the Executive Council
+
+The Executive Director may request feedback
+from the Executive Council on significant or public-facing decisions; these types of decisions do not require a formal vote. These discussions should
+occur in the **executive-council** Slack channel to ensure transparency and
+collaborative input. Examples include:
+
+* Significant public-facing changes to the pyOpenSci website.
+* Decisions impacting staff or human resources (e.g., hiring or termination).
+* Operational policies for staff (unless they carry major financial implications).
+* Strategic decisions about pursuing proposals or writing grants.
+* Substantial changes to pyOpenSci programs.
+* Strategic decisions about pursuing proposals or writing grants.
+* Substantial changes to pyOpenSci programs.
+
+By maintaining open communication, the Executive Council and Executive Director ensure that pyOpenSci’s
+decisions align with its mission and strategic goals.
+
+### Lazy consensus for decisions that don't require a vote
+
+Lazy consensus is applied for decisions such as those listed above that do not require a formal vote.
+
+Lazy consensus means that decisions are approved by default unless a member of
+the Executive Council explicitly raises an objection within a predefined period.
+This allows decisions to move forward efficiently while providing an
+opportunity for discussion.
+
+When objections are raised, further discussion occurs. This ensures decisions reflect the collective agreement of the Council
+while minimizing unnecessary delays.
+
+
+## How the Executive Council meets and communicates
```

Comment:
And then after adding lazy consensus move this to an operations  section of the guide - maybe at the bottom of this page --

add a callout - everything below is just operations and can be changed as need be. everything above is voted on and requires a vote to change

Reply:
moved